mohammad/CSS/CSSBoxModel/style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+#sq1-1 {
+  background-color: red;
+  width: 40px;
+  height: 40px;
+  display: inline-block;
+  margin: 4px;
+}
+#sq1-2 {
+  background-color: red;
+  width: 40px;
+  height: 40px;
+  display: inline-block;
+  margin: 4px;
+}
```

Comment:
Why don't use classes instead of ids when you have exactly the same styles?

Reply:
You are right
thank you